### PR TITLE
1201 exact distincts

### PR DESCRIPTION
**Pull request by derekjn**

### CODE REVIEW

File: src/test/regress/expected/cont_set_agg.out [hunk 1]
```diff
@@ -0,0 +1,65 @@
+CREATE CONTINUOUS VIEW test_set_agg0 AS SELECT set_cardinality(set_agg(x::integer)), exact_count_distinct(x) FROM test_set_agg_stream;
```
Comment (usmanm) [inline src/test/regress/expected/cont_set_agg.out]: Can we explicitly check if this works when hash values collide?

File: src/backend/utils/adt/setfuncs.c [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+#include "utils/datum.h"
+#include "utils/hsearch.h"
+#include "utils/setfuncs.h"
+#include "utils/typcache.h"
+
+#define MURMUR_SEED 0x02cffb4c45ee1fb8L
+
+typedef struct SetKey
+{
+	uint32 k;
```
Comment (usmanm) [inline src/backend/utils/adt/setfuncs.c]: Maybe rename `k` and `d` to `hash` and `value`.

**Comment by usmanm** [comment]
Looks good!

File: src/test/regress/expected/cont_set_agg.out (hunk 1 above)
Comment (derekjn) [inline src/test/regress/expected/cont_set_agg.out]: I checked manually by hard-coding a collision, might take a bit to find a murmur collision to use but I can do it.

**Pull request closed without merging**